tools/travis-linux.sh

```diff
@@ -43,7 +43,16 @@ set_up_tests() {
 run_tests() {
     npm run test-selenium
     npm run test-integration-travis
+
     # Note: Cargo recompiles every dependency with dead code. That's why
     # this step is currently the last
+
+    echo "run_tests: Running crate tests..."
+    for DIR in $(ls -d components/*); do
```

Comment:
It's usually a [bad practice](http://mywiki.wooledge.org/BashGuide/Practices#Don.27t_Ever_Do_These ) to use the output of `ls` mainly because of spaces in file names.